clean-logs.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+DIRECTORY=${AIRFLOW_HOME:-/usr/local/airflow}
+RETENTION=${ASTRONOMER__AIRFLOW__WORKER_LOG_RETENTION_DAYS:-15}
```

Comment:
```suggestion
RETENTION=${AIRFLOW__LOG_RETENTION_DAYS:-15}
```

This file won't be included in the apache Docker image without more work being done (cos the prod images do `pip install apache-airflow==...`) so putting this file here won't help/do much by itself.

Reply:
Oh you added a COPY command, missed that.

Still, should probably move this somewhere else in the repo, say scripts/

Reply:
Yeah. was just hacking to get it to work first.